.circleci/Dockerfile

```
@@ -1,32 +1,35 @@
 FROM golang:1.13.1
 
-## Warm apt cache
-RUN apt-get update
+# Warm apt cache
+RUN apt-get update && \
+    apt-get install -y wget unzip \
+    openjdk-11-jre \
+    bzip2
 
 # Install swagger-codegen
-ENV SWAGGER_CODEGEN_VERSION=2.2.2
-RUN apt-get install -y openjdk-11-jre wget && \
-    wget http://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER_CODEGEN_VERSION}/swagger-codegen-cli-${SWAGGER_CODEGEN_VERSION}.jar \
-    -O /usr/local/bin/swagger-codegen-cli.jar && \
-    apt-get remove -y wget
-ENV SWAGGER_CODEGEN="java -jar /usr/local/bin/swagger-codegen-cli.jar"
+ENV SWAGGER_CODEGEN_VERSION=2.4.8
+RUN wget http://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER_CODEGEN_VERSION}/swagger-codegen-cli-${SWAGGER_CODEGEN_VERSION}.jar \
+    -O /usr/local/bin/swagger-codegen-cli.jar
+
+# Wrap the jar for swagger-codgen
+RUN echo -e '#!/bin/bash\n java -jar /usr/local/bin/swagger-codegen-cli.jar "$@"' > /usr/local/bin/swagger-codegen && \
```

```suggestion
RUN echo -e '#!/bin/bash\njava -jar /usr/local/bin/swagger-codegen-cli.jar "$@"' > /usr/local/bin/swagger-codegen && \
```
What monster puts a single space at the start of a new line.